mule/sign.sh

```diff
@@ -21,7 +21,7 @@ make_hashes () {
     for ext in ${EXTENSIONS[*]}
     do
         if ls *"$VERSION"*."$ext"; then
-            HASHFILE="hashes_${OS_TYPE}_${ARCH}_${VERSION}_${ext}"
+            HASHFILE="hashes_${OS_TYPE}_${ARCH}_${VERSION}_${ext/./}"
```

Comment:
Stylistically, I think replacing it with underscore makes it match the deb hashfiles:

```suggestion
            HASHFILE="hashes_${OS_TYPE}_${ARCH}_${VERSION}_${ext/./_}"
```

Reply:
I made a change so it replaces all `.` with `_` in the extensions. `HASHFILE="hashes_${OS_TYPE}_${ARCH}_${VERSION}_${ext//./_}"`